Duplicati/Library/Backend/Storj/StorjBackend.cs

```diff
@@ -84,52 +93,52 @@ public Tardigrade(string url, Dictionary<string, string> options)
         {
             InitStorjLibrary();
 
-            var auth_method = options[TARDIGRADE_AUTH_METHOD];
+            var auth_method = options[STORJ_AUTH_METHOD] ?? options[TARDIGRADE_AUTH_METHOD];
```

Comment:
I took an existing Tardigrade backup configuration, updated to this branch, and then attempted to run a backup.  This resulted in the following exception:

![image](https://user-images.githubusercontent.com/17345532/116898328-169d3d80-abeb-11eb-96f6-b50a8751756e.png)

Should we use the `TryGetValue` pattern here instead?

Reply:
I switched to "TryGetValue". May you verify this again? Looks good from my side - but I also said that on the previous try... :)